### ISSUE TITLE
Svelte mode does not support built-in treesitter

### ISSUE THREAD

**Issue by tiberiuiurco** [leafOfTree/svelte-mode#15]
Svelte mode does not support built-in treesitter.
I do not think it would be hard to implement.
For example, instead of typescript-mode, it should use typescript-ts-mode.
Most of the people now are switching to builtin treesitter, so think functionality will be very good for compatibility.

**Comment by leafOfTree** [comment]
I agree that we need a treesitter svelte mode. However, there seem to be many differences between a regular mode and a treesitter mode. I'm trying a new `svelte-ts-mode`.

**Comment by LazerJesus** [comment]
any progress on this? 
A quickfix for me would be if scripts with ts would use `typescript-ts-mode` instead of `typescript-mode`

**Comment by leafOfTree** [comment]
Does simply using `typescript-ts-mode` instead of `typescript-mode` work for you?

Last time I checked it seem much complicated. So I started a new mode: https://github.com/leafOfTree/svelte-ts-mode. You can give it a try.
